pkg/cmd/grafana-cli/runner/wire.go

```diff
@@ -236,8 +236,10 @@ var wireSet = wire.NewSet(
 	pushhttp.ProvideService,
 	plugincontext.ProvideService,
 	contexthandler.ProvideService,
-	jwt.ProvideService,
-	wire.Bind(new(models.JWTService), new(*jwt.AuthService)),
+	jwt.ProvideVerificationService,
+	jwt.ProvideUserAuthService,
```

Comment:
i'm not clear on the difference between `pkg/cmd/grafana-cli/runner/wire.go` and `pkg/server/wire.go`. should i be adding the new service to both?

Reply:
You might need to depending on dependencies dependent on your service used within the grafana-cli